File: src/kyu_7/sort_the_gift_code/python/sort_the_gift_code_test.py
import codewars_test as test
from solution import sort_gift_code

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(sort_gift_code('abcdef'), 'abcdef')
        test.assert_equals(sort_gift_code('pqksuvy'), 'kpqsuvy')
        test.assert_equals(sort_gift_code('zyxwvutsrqponmlkjihgfedcba'), 'abcdefghijklmnopqrstuvwxyz')

File: src/kyu_7/sort_the_gift_code/csharp/sort_the_gift_code_test.cs
namespace Solution
{
  using NUnit.Framework;
  using System;

  [TestFixture]
  public class SolutionTest
  {
    [Test]
    public void SampleTest()
    {
      Assert.AreEqual("abcdef", Kata.SortGiftCode("abcdef"));
      Assert.AreEqual("kpqsuvy", Kata.SortGiftCode("pqksuvy"));
      Assert.AreEqual("abcdefghijklmnopqrstuvwxyz", Kata.SortGiftCode("zyxwvutsrqponmlkjihgfedcba"));
    }
  }
}